clients/src/main/java/org/apache/kafka/clients/NetworkClient.java

```diff
@@ -515,6 +527,7 @@ private void doSend(ClientRequest clientRequest, boolean isInternalRequest, long
             completeResponses(responses);
             return responses;
         }
+        ensureActive();
```

Comment:
Was there a strong reason to check aborted sends before this?

Reply:
Not really, moved it to the start of the method.